calico_containers/adapter/netns.py

```diff
@@ -192,7 +192,7 @@ def set_up_endpoint(ip, cpid, next_hop_ips,
 
     # Return an Endpoint.
     network = IPNetwork(IPAddress(ip))
-    ep = Endpoint(ep_id=ep_id, state="active", mac=mac, if_name=veth_name)
+    ep = Endpoint(hostname, "docker", cpid, endpoint_id, "active", mac)
```

Comment:
I think for now hardcoding to docker will be fine.  However, add a TODO comment, and add an item to the google doc so that we don't forget to fix this up.

This will probably be changed by Toms libnetwork stuff too.